e2e-tests/trade-finance-manager/cypress/fixtures/dateConstants.js

```diff
@@ -0,0 +1,96 @@
+import {
+  sub, add, format, getUnixTime,
+} from 'date-fns';
+
+const today = new Date();
```

Comment:
we already have this file in e2e-tests/gef/etc

Can we move that file inside `e2e-tests/e2e-fixtures/` and then import it where we need it? this to avoid duplicating it in multiple locations

Reply:
done